app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -157,18 +160,15 @@ class ReportFilter extends React.Component {
                   </div>
 
                   <div className="filter-values">
-                    <p>
-                      <input type="checkbox" className="filled-in" id="bacteria" defaultChecked="checked" />
-                      <label htmlFor="bacteria">Bacteria</label>
-                    </p>
-                    <p>
-                      <input type="checkbox" className="filled-in" id="fungi" defaultChecked="checked" />
-                      <label htmlFor="fungi">Fungi</label>
-                    </p>
-                    <p>
-                      <input type="checkbox" className="filled-in" id="virus" defaultChecked="checked" />
-                      <label htmlFor="virus">Virus</label>
-                    </p>
+                    { this.all_categories.map((category, i) => {
+                      return (
+                        <p key={i}>
+                          <input type="checkbox" className="filled-in" id={category.name} defaultChecked="checked" />
```

Comment:
i think the box shouldn't be checked by default. It should reflect what people selected.